### PR TITLE
Add pagination tests

### PR DESCRIPTION
**Pull request by codiebeulaine**
## Purpose
_Creating tests for message pagination and fix flake8_

### CODE REVIEW

File: contentrepo/settings/base.py [hunk 1]
```diff
@@ -145,8 +147,10 @@
     ('es', "Spanish"),
 ]
 
-# Base URL to use when referring to full URLs within the Wagtail admin backend -
-# e.g. in notification emails. Don't include '/admin' or a trailing slash
+# Base URL to use when referring to full URLs within the
+# Wagtail admin backend -
+# e.g. in notification emails. Don't include '/admin'
+# or a trailing slash
 BASE_URL = 'http://example.com'
```
Comment (KaitCrawford) [inline contentrepo/settings/base.py:154]: This should probably be an actual url or pull from env vars

File: home/tests/test_api.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        content = json.loads(response.content)
+        self.assertEquals(
+            content, {'detail': 'The requested message does not exist'})
+
+        # it should return an appropriate error if requested message is not
+        # a positive integer value
+        response = self.client.get(
+            "/api/v2/pages/4/?whatsapp=True&message=notint")
+        content = json.loads(response.content)
+        self.assertEquals(
```
Comment (KaitCrawford) [inline home/tests/test_api.py:57]: does this return a http 400 status code?

**Review by KaitCrawford — APPROVED** [review]
:+1: minor comments/questions that don't have to affect this PR

I think it would also be good to return the number of messages for the whole page. so we can display "page 3 of 6" or whatever

File: home/tests/test_api.py (hunk 2 above)
Comment (codiebeulaine) [reply home/tests/test_api.py:60]: it does now

**Review by KaitCrawford — APPROVED** [review]
:+1: :star2: :cake:

**Pull request merged**